cmd/nic-configuration-daemon/main.go

```diff
@@ -31,6 +33,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	err = maintenanceoperator.AddToScheme(scheme)
+	if err != nil {
+		log.Log.Error(err, "failed to load maintenance operator to scheme")
+		os.Exit(1)
+	}
```

Comment:
nit:

```suggestion
	utilruntime.Must(maintenanceoperator.AddToScheme(scheme))
```

with `import utilruntime "k8s.io/apimachinery/pkg/util/runtime"`